_libraries/ASP-NET-Web-API-OData.md

```diff
@@ -2,11 +2,11 @@
 category: net
 name: ASP.NET Web API OData
```

Comment:
we should add an entry for ASP.NET Core OData

Reply:
The link for Web API forwards to https://learn.microsoft.com/en-us/odata/. Do we want that to be the link for ASP.NET Core OData as well, or something else?

Reply:
Let's use https://learn.microsoft.com/en-us/odata/webapi-8/Overview for Asp.net Core OData, and https://learn.microsoft.com/en-us/odata/webapi/getting-started for Web API.

Reply:
I don't see the new link for ASP.NET Core OData -- is that in a new file that needs to be added to the PR?